src/components/LogInModal/styles.module.scss

```diff
@@ -0,0 +1,66 @@
+@import 'scss/colors';
+@import 'scss/mixins';
+@import 'scss/variable';
+
+.container{
+    @include flex(flex-start, center, column);  
+    background: $white;
+    border-radius: 4px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.5);
```

Comment:
Add shadow as a variable, pretty sure it'll be useful later

Reply:
done